### PR TITLE
[Release 3.0.1] Merging to Master Branch 

### PR DESCRIPTION
**Pull request by adityagajbhiye**
- Renamed the Document folder to DOCS
- Kics issue in Docker and deployment file resolved.
- Table sorting resolved.
- Font assets from client repo to avoid breaking.
- Readme.md document updated.
- License and Readme in charts updated.
- Provider page title added, Reusable DownloadCSV component.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,9 +6,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
```
Comment (Siegfriedk) [inline CHANGELOG.md:7]: Version update is missing?

File: Docs/images/argocd-apps.png
Comment (Siegfriedk) [inline Docs/images/argocd-apps.png:1]: it should be docs not Docs folder

File: charts/dftfrontend/README.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+/charts/dft-frontend/templates/deployment.yaml	Template to configure your application deployment.
+/charts/dft-frontend/templates/ingress.yaml	Template to configure your application deployment.
+/charts/dft-frontend/templates/service.yaml	Template to configure your application deployment.
+/charts/dft-frontend/templates/hpa.yaml	Template to configure your application deployment.
+/charts/dft-frontend/templates/NOTES.txt	Helper to enable locating your application IP and PORT
+
+Helm Commands
+$ helm repo add eclipse-tractusx-dft-frontend https://github.com/eclipse-tractusx/dft-frontend/tree/main/charts/dft-frontend
+
+$ helm install my-release eclipse-tractusx/dft-frontend --version 1.8.1
```
Comment (Siegfriedk) [inline charts/dftfrontend/README.md:33]: Version is wrong and i believe you renamed the helm chart to dftfrontend?

File: charts/dftfrontend/values.yaml [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  endpoints:
+    ## Default api exposing health checks etc
+    default:
+      # -- The network port, which the "default" api is going to be exposed by the container, pod and service
+      port: "80"
+      # -- The path mapping the "default" api is going to be exposed at
+      path: /  
+          
+ingress:
+  - enabled: true
```
Comment (Siegfriedk) [inline charts/dftfrontend/values.yaml:52]: ingress should be disabled by default

**Review by Siegfriedk — CHANGES_REQUESTED** [review]
Was unable to verify this but you also need to add the font you added as a dependency ;

In this project it was added to the NOTICE file 3th party dep: https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/blob/main/NOTICE.md

File: charts/dftfrontend/values.yaml (hunk 3 above)
Comment (adityagajbhiye) [reply charts/dftfrontend/values.yaml:52]: Set to default ingress

File: charts/dftfrontend/README.md (hunk 2 above)
Comment (adityagajbhiye) [reply charts/dftfrontend/README.md:33]: version updated

File: Docs/images/argocd-apps.png
Comment (adityagajbhiye) [reply Docs/images/argocd-apps.png:1]: Docs renamed to docs folder

File: CHANGELOG.md (hunk 1 above)
Comment (adityagajbhiye) [reply CHANGELOG.md:7]: Version update and change log update.

**Comment by ajithsimons** [comment]
@Siegfriedk  NOTICE file have the third party font dependency details already.

**Pull request merged**